packages/api/helper/mailer.js

```diff
@@ -84,7 +84,7 @@ function verifyHcEmail(nomination) {
       locals: {
         name: nomination.providerName,
         appUrl: process.env.APP_URL,
-        urlLink: `${process.env.APP_URL}/confirmation/${emailToken}`,
+        urlLink: `${process.env.APP_URL}/api/confirmation/${emailToken}`,
```

Comment:
It's a bit of an unconventional flow for the user to be directed to an API link directly. I think this should probably redirect to a React page on the app and the app should make the API call when the page is loaded.

Reply:
So I changed the link that the user clicks on to be a react page and then once the react page renders an api call is made to update the emailValidated property